<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mxpackageinstaller</class>
 <widget class="QDialog" name="mxpackageinstaller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MX Package Installer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="buttonBar" columnstretch="0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Display help </string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/gnome-help.png</normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/gnome-help.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelMX">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../../../../usr/share/mx-packageinstaller/icons/mxfcelogo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="buttonInstall">
       <property name="toolTip">
        <string>Install selected packages</string>
       </property>
       <property name="text">
        <string>Install</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/dialog-ok.png</normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/dialog-ok.png</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="buttonCancel">
       <property name="toolTip">
        <string>Cancel any changes then quit</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/gtk-cancel.png</normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/gtk-cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonAbout">
       <property name="toolTip">
        <string>About this application</string>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/accessories-text-editor.png</normaloff>../../../../../../../../usr/share/mx-packageinstaller/icons/accessories-text-editor.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+B</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="selectionPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Package</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Info</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Install additional packages&lt;/span&gt;&lt;/p&gt;&lt;p&gt;To use this feature, make sure you are connected to the internet.&lt;/p&gt;&lt;p&gt;Greyed out items have already been installed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="yesCheckBox">
         <property name="text">
          <string>Use option --yes for apt-get install</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="outputPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="buttonDetails">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Show details</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="outputBox">
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>pressed()</signal>
   <receiver>mxpackageinstaller</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
